src/keyboard.rs

```diff
@@ -1709,7 +1700,7 @@ bitflags! {
         /// The "alt" key.
         const ALT = 0b100 << 6;
         /// This is the "windows" key on PC and "command" key on Mac.
-        const SUPER = 0b100 << 9;
+        const META = 0b100 << 9;
```

Comment:
This breaks user configs all along, because a change in serialization format. If someone relied on winit here they are kind of screwed.

Reply:
I kept the `const SUPER` around as a deprecated alias, that should make the `serde` impl accept `SUPER` as well.